easybuild/easyconfigs/q/Qt6/Qt6-6.2.3-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,74 @@
+easyblock = 'ConfigureMake'
```

Comment:
We used to have a dedicated EasyBlock for Qt: https://github.com/easybuilders/easybuild-easyblocks/blob/develop/easybuild/easyblocks/q/qt.py Now, I found that the build system for Qt6 changed to CMake https://www.qt.io/blog/qt-6-build-system so it makes sense that we can't use the old EasyBlock as it was. 

However, I think we should make a new one, since a) the build procedure seems to be pretty peculiar and b) we don't want to loose the 'good stuff' from the previous EasyBlock. E.g. automatic selection of the target platform that was done [here](https://github.com/easybuilders/easybuild-easyblocks/blob/745740a15232c7609288d661b1c9a687e3b31de3/easybuild/easyblocks/q/qt.py#L67), custom sanity check step [here](https://github.com/easybuilders/easybuild-easyblocks/blob/745740a15232c7609288d661b1c9a687e3b31de3/easybuild/easyblocks/q/qt.py#L171), etc. We'll need to figure out what's still relevant for Qt6, as not everything might be.

If I understand correctly, the build procedure is something like

```
configure <qt_arguments> -- <CMake_arguments>
cmake --build .
cmake --install .
```
right? Isn't the old syntax of `configure, make, make install` also not still supported then? It might be easier to use that, since that is already used by the `ConfigureMake` EasyBlock (from which the `Qt` EasyBlock inherits)

I think it would make most sense to simply expand the old Qt EasyBlock and use version checks (e.g. like [this](https://github.com/easybuilders/easybuild-easyblocks/blob/745740a15232c7609288d661b1c9a687e3b31de3/easybuild/easyblocks/q/qt.py#L92)) to make it behave differently for Qt6. You'll need to figure out which parts of the Qt5 EasyBlock are no longer applicable to the Qt6 build (e.g. I think setting the `QMAKE` variables is no longer needed?).

Reply:
I have Opened pull request: https://github.com/easybuilders/easybuild-easyconfigs/pull/17902 with newer compilers and all dependencies includes. 
Build procedure with CMake is quite stable and no longer a problem as it was with Qt5. Qt5 EasyBlock are no longer applicable to the Qt6 build.